DOCS-2987: Add PrestaShop FAQ article about translating payment methods (#1971)
MultiSafepay/docs: content/integration/ready-made/prestashop-1-7/faq/translating-payment-methods.md

---

> title : "Translating the module"

Reviewer: ```suggestion
title : "Translating the MultiSafepay module"
```

> meta_title: "PrestaShop 1.7 - Translating the module - MultiSafepay Docs"

Reviewer: ```suggestion
meta_title: "PrestaShop 1.7 - Translating the MultiSafepay module - MultiSafepay Docs"
```

read_more: "."

> url: '/prestashop-1-7/translating-the-module/'

Reviewer: ```suggestion
url: '/prestashop-1-7/translating-multisafepay-module/'
```

---

> The PrestaShop 1.7 plugin offers the possibility to texts that belongs to the module.

Reviewer: ```suggestion
To translate elements of the MultiSafepay module in the PrestaShop 1.7 plugin, follow these steps:
```

> To translate the module:

Reviewer: ```suggestion
```

1. Sign in to your [backend](/glossaries/multisafepay-glossary/#backend).
2. Go to **Improve** > **International** > **Translations**.

> 3. Under the **Modify translations** tab select **Installed module translations** for the **Type of translation** field.

Reviewer: ```suggestion
3. Under the **Modify translations** tab, in the **Type of translation** field, select **Installed module translations**.
```

4. Select the **MultiSafepay** module.
5. Select the **Language**.

> 6. Press on **Modify**.

Reviewer: ```suggestion
6. Click **Modify**.
```

> 7. On this page you can make the required translations

Reviewer: ```suggestion
7. Enter the required translations.
```

Reviewer: @Daniel-MultiSafepay Can you send a screenshot of the page? Do you need to click **Save** or something after entering the translations?

> 8. Press **Save** button. 

Reviewer: ```suggestion
8. Click **Save**. 
```

> In the specific case of the translations of the names of the payment methods:

Reviewer: ```suggestion
To translate the names of payment methods:
```

1. Go to **Improve** > **MultiSafepay** > **Payment methods**.

> 2. Select the Payment Method you would like to translate.

Reviewer: ```suggestion
2. Select the payment method you want to translate.
```
